adoc/deployment-preparation.adoc

```diff
@@ -8,42 +8,35 @@ and must be maintained on this machine, but it is not a member of the skuba clus
 [[ssh.configuration]]
 === Basic SSH Key Configuration
 
+*In order to successfully deploy {productname}, you need to have an SSH agent.* This is important, because it is required in order to use the installation tools `skuba` and `terraform`.
+
 [NOTE]
 ====
 The use of `ssh-agent` comes with some implications for security that you should take into consideration.
 
 link:http://rabexc.org/posts/pitfalls-of-ssh-agents[The pitfalls of using ssh-agent]
 
 To avoid these risks please make sure to either use `ssh-agent -t <TIMEOUT>` and specify a time
-after which the agent will self-terminate or terminate the agent yourself before logging out by running
-`ssh-agent -k`.
+after which the agent will self-terminate or terminate the agent yourself before logging out by running `ssh-agent -k`.
 ====
 
 To log in to the created cluster nodes from the Management machine, you need to configure an SSH key pair.
-This key pair needs to be trusted by the user you will log in as upon each cluster node; this user defaults to
-"sles".  In order to use the installation tools `terraform` and `skuba`, this trusted keypair also must be
-loaded into the SSH agent.
+This key pair needs to be trusted by the user you will log in as into each cluster node; this user defaults to "sles".
```

Comment:
```suggestion
This keypair needs to be trusted by the account you will log in with on each cluster node; this defaults to "sles".
```
Both yours and mine are a bit awkward.  How about this one? :D 

Related - do we use "account" or "user" or "username"?  Probably oughtta be consistent. :)

Reply:
I'm pretty sure we mostly use "user", but I like "user account", so I would go with that :D and I also did a search for "key pair" and "keypair", it seems that the former us used more frequently, "keypair" is actually identified as a spelling mistake by my spellchecker.

Reply:
"Key Pair" as two words is how the openssh man page describes it, but usually they just say "key." So that'll be my answer as well going forward. :)  Good catch; referring to the pair is an an old habit of mine I should probably break.